### ISSUE TITLE
 in 《Modal sensor》--s1=lsqcurvefit('fun',a01,x0,y1);

### ISSUE THREAD

**Issue by ZhenhongDu** [WeisongZhao/AdaptiveOptics.simulation#1]
in 《Modal sensor》--s1=lsqcurvefit('fun',a01,x0,y1);
Maybe I didn't find the definition of this function ‘fun’

**Comment by ZhenhongDu** [comment]
Is it a quadratic function

**Comment by WeisongZhao** [comment]
True, it should be a quadratic function.

**Comment by ZhenhongDu** [comment]
But sometimes the metric value do not like a quadratic function,It's more like a primary function（Linear increase）

**Comment by WeisongZhao** [comment]
YES. It would be not stable once the aberrations are larger than ~0.7 rad, but the overall tendency is true. Thus, we can use the `Modal_sensor_new` strategy to pre correct the large components (> 0.7 rad).
